documentation/reference/search-definitions-reference.html

```diff
@@ -2644,20 +2637,29 @@ <h2 id="field_types">Field types</h2>
 
 <tr><th id="type:reference">reference&lt;document-type&gt;</th>
 <td>
+<p>
 A <em>reference&lt;document-type&gt;</em> field is a reference to an instance of a document-type -
-i.e. a foreign key. The reference is the <a href="../documents.html">document id</a>
-of the document-type instance, hence a string.
-References are used to join documents in a
-<a href="../search-definitions.html#document-references">parent-child relationship</a>.
+i.e. a foreign key.
+The reference is the <a href="../documents.html">document id</a> of the document-type instance, hence a string attribute.
```

Comment:
I think attribute should be dropped here as we are talking about the string value. The reference field is not a string attribute.

Reply:
you are right - corrected